internal/httphandler/utils.go

```diff
@@ -195,6 +196,28 @@ func extractVerifiedQueryPayload(w http.ResponseWriter, r *http.Request, queryTy
 			Id:      params["id"],
 			Version: version,
 		}
+	case constants.PostDataQuery:
+		if r.Body == nil {
+			httputils.SendHTTPResponse(w, http.StatusBadRequest, &types.HttpResponseErr{ErrMsg: "query is empty"})
+			return nil, true
+		}
+
+		b, err := io.ReadAll(r.Body)
```

Comment:
how does it passes the CI then? 

This is because the CI uses 1.16 since we moved to GitHub. I remember updating my local version. I have updated the go.mod to 1.16. We should in fact bump it to 1.17 soon

Reply:
I will update mine as well